clustered/server/src/main/java/org/ehcache/clustered/server/internal/messages/EhcacheDataSyncMessage.java

```diff
@@ -20,17 +20,17 @@
 
 import com.tc.classloader.CommonComponent;
 
+import java.util.Map;
+
 @CommonComponent
 public class EhcacheDataSyncMessage extends EhcacheSyncMessage {
 
   private final String cacheId;
-  private final long key;
-  private final Chain chain;
+  private final Map<Long, Chain> chainMap;
 
-  public EhcacheDataSyncMessage(final String cacheId, final long key, final Chain chain) {
+  public EhcacheDataSyncMessage(final String cacheId, final Map<Long, Chain> chainMap) {
     this.cacheId = cacheId;
-    this.key = key;
-    this.chain = chain;
+    this.chainMap = chainMap;
```

Comment:
Should probably be wrapped in `Collections.unmodifiableMap()` for peace of mind.